src/paper/tasks.py

```diff
@@ -1063,11 +1029,11 @@ def pull_papers(start=0, force=False):
 NUM_DUP_STOP = 30
 
 # Pull Daily
-@periodic_task(
-    run_every=crontab(minute=0, hour='*/6'),
-    priority=1,
-    options={'queue': f'{APP_ENV}_autopull_queue'}
-)
+# @periodic_task(
```

Comment:
This function returns immediately. I'm commenting out the periodic part to avoid celery picking it up